README.md

```diff
@@ -138,20 +138,20 @@ assert!(subimg.dimensions() == (100, 100));
 These are the functions defined in the `imageops` module. All functions operate on types that implement the `GenericImage` trait.
 
 + **blur**: Performs a Gaussian blur on the supplied image.
-+ **brighten**: Brighten the supplied image
-+ **huerotate**: Hue rotate the supplied image by degrees
-+ **contrast**: Adjust the contrast of the supplied image
-+ **crop**: Return a mutable view into an image
++ **brighten**: Brighten the supplied image.
++ **huerotate**: Hue rotate the supplied image by degrees.
++ **contrast**: Adjust the contrast of the supplied image.
++ **crop**: Return a mutable view into an image.
 + **filter3x3**: Perform a 3x3 box filter on the supplied image.
-+ **flip_horizontal**: Flip an image horizontally
-+ **flip_vertical**: Flip an image vertically
-+ **grayscale**: Convert the supplied image to grayscale
++ **flip_horizontal**: Flip an image horizontally.
++ **flip_vertical**: Flip an image vertically.
++ **grayscale**: Convert the supplied image to grayscale.
 + **invert**: Invert each pixel within the supplied image This function operates in place.
-+ **resize**: Resize the supplied image to the specified dimensions
++ **resize**: Resize the supplied image to the specified dimensions. Note that this is very slow in debug mode. Make sure to use release mode when using this.
```

Comment:
I believe the slowness is not limited to `resize`, it probably affects most of these methods as they all make use of quite a few layers of abstractions. Maybe append this to the paragraph in line 138?

Reply:
Agreed, pushed another update